### PR TITLE
Add Customers Listing Page

### PR DESCRIPTION
**Pull request by faxblaster**
Adds a simple listing page and navigation item for Customers.  Also includes a minor refactor of pagination to make it easier to use across multiple components. Fixes some formatting with prettier.

Screen Shot 2021-07-21 at 8 58 49 AM

### CODE REVIEW

**Comment by faxblaster** [comment]
Leaving in draft until we can merge #30 & #32.

File: resources/js/components/CustomersTable.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                        <td>{{ customer.email }}</td>
+                        <td>
+                            <span v-if="customer.currency">
+                                {{ customer.currency | money(customer.balance) }}
+                            </span>
+                            <span v-else>-</span>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
```
Comment (mattstauffer) [inline resources/js/components/CustomersTable.vue:47]: We might want to add an empty state that just says something like "No customers"

File: resources/js/components/PaginationLinks.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
```
Comment (mattstauffer) [inline resources/js/components/PaginationLinks.vue:1]: Wow, easy refactor. Nice work!

**Review by mattstauffer — CHANGES_REQUESTED** [review]
Great work! Only one requested change

**Pull request merged**